rel/fed1.vars.config

```diff
@@ -1,9 +1,9 @@
-{hosts, "[ \"fed1\" ]"}.
+{hosts, "[ \"fed1\", \"localhost\" ]"}.
 {sm_backend, "{mnesia, []}"}.
 {auth_method, "internal"}.
 {outgoing_s2s_port, 5269}.
 {odbc_server, ""}.
-{s2s_addr, "{ {s2s_addr, \"localhost\"}, {127,0,0,1} }."}.
+{s2s_addr, "{ {s2s_addr, \"localhost\"}, {127,0,0,1} }. { {s2s_addr, \"localhost.bis\"}, {127,0,0,1} }."}.
```

Comment:
We already had 2 localhosts, where `"localhost.bis"` is a (one of) domain specific to the first cluster and "`fed1`" to the second one. Because I changed `"localhost"` to be a shared domain for global distrib testing purposes, I also changed S2S tests' user to use `"localhost.bis"` address.

Reply:
Looks like an epic hack.
It would break other S2S tests, if we had more. For example, MUC tests with S2S.